frontends/web/src/components/devices/bitbox02/settings.tsx

```diff
@@ -161,6 +163,13 @@ class Settings extends Component<Props, State> {
                         </Footer>
                     </div>
                 </div>
+                <Guide>
+                    <Entry key="guide.manageBB02.backups" entry={t('guide.manageBB02.backups')} />
+                    <Entry key="guide.manageBB02.seed" entry={t('guide.manageBB02.seed')} />
+                    <Entry key="guide.manageBB02.why_seed" entry={t('guide.manageBB02.why_seed')} />
+                    <Entry key="guide.manageBB02.what_passphrase" entry={t('guide.manageBB02.what_passphrase')} />
```

Comment:
we use camelCase, not snake_case: `whatPassphrase`, `whyPassphrase`

Reply:
done

Reply:
done